src/main/java/frc/trigon/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,74 @@
+package frc.trigon.robot.subsystems.intake;
+
+import com.ctre.phoenix6.controls.StaticBrake;
+import com.ctre.phoenix6.controls.VoltageOut;
+import frc.trigon.robot.constants.OperatorConstants;
+import frc.trigon.robot.subsystems.MotorSubsystem;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+
+public class Intake extends MotorSubsystem {
+    private final TalonFXMotor
+            masterMotor = IntakeConstants.MASTER_MOTOR,
+            followerMotor = IntakeConstants.FOLLOWER_MOTOR;
+    private final VoltageOut voltageRequest = new VoltageOut(0).withEnableFOC(IntakeConstants.FOC_ENABLED);
+    private final StaticBrake staticBrakeRequest = new StaticBrake();
+    private IntakeConstants.IntakeState targetState;
+
+    public Intake() {
+        setName("Intake");
+    }
+
+    @Override
+    public void periodic() {
+        masterMotor.update();
+        IntakeConstants.DISTANCE_SENSOR.updateSensor();
+        updateMechanism();
+    }
+
+    @Override
+    public void setBrake(boolean brake) {
+        masterMotor.setBrake(brake);
+        followerMotor.setBrake(brake);
+    }
```

Comment:
We don't implement this for spinning wheels.

Reply:
why?

Reply:
Because we don't want them to brake. Every motor that implements this function brakes on teleop. But we just want the shooter to always be in coast